code-for-canada/project-thundercat
# Accessibility - Update HTML Lang property

# Description

Update HTML Lang attribute on toggling the language. This is important to browser tools and accessibility.

I've installed React Helmet to allow us to do this which will also enable things like updating the title when the page changes: https://github.com/nfl/react-helmet

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## Screenshot

![engfr](https://user-images.githubusercontent.com/4640747/56915047-8cfa8880-6a83-11e9-8eb0-c74618827711.gif)


# Testing

Manual steps to reproduce this functionality:

1.  Open the dev tools
2.  Toggle the language and notice the HTML lang attribute updating.

# Checklist

Applicable for all code changes.

I'm unable to test this on IE because I'm on my macbook.
